python/ccxt/base/exchange.py

```diff
@@ -707,7 +707,7 @@ def handle_http_status_code(self, http_status_code, http_status_text, url, metho
         string_code = str(http_status_code)
         if string_code in self.httpExceptions:
             Exception = self.httpExceptions[string_code]
-            raise Exception(' '.join([self.id, method, url, string_code, http_status_text, body]))
+            raise Exception(self.id + ' '.join([self.id, method, url, string_code, http_status_text, body]))
```

Comment:
Now we're prepending `self.id` here twice, this line concatenates the necessary parts, i guess, we should revert it?